src/test/ui/lint/lint-ctypes.rs

```diff
@@ -10,7 +10,7 @@
 
 #![deny(improper_ctypes)]
 #![feature(libc)]
-
+#![allow(private_in_public)]
```

Comment:
I was unsure why the error changed and why i needed this. Also is there is a better way to handle this?